src/tools/cephfs/DataScan.cc

```diff
@@ -33,14 +33,16 @@ void DataScan::usage()
 {
   std::cout << "Usage: \n"
     << "  cephfs-data-scan init [--force-init]\n"
-    << "  cephfs-data-scan scan_extents [--force-pool] <data pool name>\n"
-    << "  cephfs-data-scan scan_inodes [--force-pool] [--force-corrupt] <data pool name>\n"
+    << "  cephfs-data-scan scan_extents [--force-pool] [--worker_n N --worker_m M] <data pool name>" << std::endl;
```

Comment:
This whole section is one statement, like `std::cout << A << B << C;`

What you're changing it to is something like `std::cout << A << B << std::endl; << C;`, it doesn't make sense.

You *could* do it by using std::endl and a trailing "<<" instead of ";", but really I'd just use "\n" like the original code.